dashboard/lib/services/lti.rb

```diff
@@ -327,5 +332,14 @@ def self.initialize_lms_landing_session(session, lti_provider_name, new_cta_type
         user_type: user_type,
       }
     end
+
+    def self.update_user_name(user, nrps_member)
```

Comment:
In Rails, the `update` method/action usually implies committing changes to the DB (i.e., calling `save`).
Since this method only assigns attributes without saving, I suggest using `assign` in the method name instead.
```suggestion
    def self.assign_user_name(user, nrps_member)
```